packages/react-storage/src/components/StorageBrowser/context/actions/listLocationsAction.ts

```diff
@@ -69,6 +70,8 @@ export const createListLocationsAction = (
         nextToken: nextNextToken,
         pageSize: remainingPageSize,
       });
+      checkRequiredKeys(output, `ListLocationsOutput`, ['locations']);
```

Comment:
What can we say about each location?

Reply:
updated to check individual location's schema